lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -326,17 +343,23 @@ public static String getMavenPropertyInRequest(IPositionRequest request) {
 		return null;
 	}
 
-	private Hover collectProperty(IPositionRequest request, String property) {
+	private Hover collectProperty(IHoverRequest request, String property) {
+		boolean supportsMarkdown = request.canSupportMarkupKind(MarkupKind.MARKDOWN);
 		DOMDocument doc = request.getXMLDocument();
 		MavenProject project = cache.getLastSuccessfulMavenProject(doc);
 		if (project != null) {
 			Map<String, String> allProps = getMavenProjectProperties(project);
+			UnaryOperator<String> toBold = message -> supportsMarkdown ? MarkdownUtils.toBold(message) : message;
 
 			for (Entry<String, String> prop : allProps.entrySet()) {
 				String mavenProperty = prop.getKey();
 				if (property.equals(mavenProperty)) {
-					return new Hover(new MarkupContent("markdown", "Property: " + mavenProperty
-							+ MavenPluginUtils.LINE_BREAK + "Value: " + prop.getValue() + MavenPluginUtils.LINE_BREAK));
+					String message = toBold.apply("Property: ") + mavenProperty 
+							+ MavenPluginUtils.LINE_BREAK 
+							+ toBold.apply("Property: ") + prop.getValue() 
+							+ MavenPluginUtils.LINE_BREAK;
```

Comment:
`MavenPluginUtils.LINE_BREAK` could probably be moved to `MarkdownUtils`

Reply:
Also, the message remains the same regardless of whether markdown is used. I'm not sure how much of an issue this is, but since MavenPluginUtils.LINE_BREAK (which is "/n/n") is used, there will be double new lines in the plaintext message. 

Maybe something like:
```Java
String lineBreak = supportsMarkdown? MavenPluginUtils.LINE_BREAK : "/n"
```
would work?